src/app/(views)/github/t/@pullButtons/pulls/page.tsx

```diff
@@ -14,12 +17,19 @@ import {
   useUpdateRepoOrPull
 } from '@/contexts/github/root/selected-pulls-provider'
 
+export enum ESettingsCode {
+  EXPAND_ALL_BUTTON = 'EXPAND_ALL_BUTTON',
+  SELECT_ALL_DEPENDABOT_PULL_REQUESTS = 'SELECT_ALL_DEPENDABOT_PULL_REQUESTS',
+  START_REVIEW_BUTTON = 'START_REVIEW_BUTTON'
+}
+
 export default function Page() {
   const { repos } = useRepos()
   const flattenCheckedPulls = getFlattenCheckedPulls(repos)
   const { translate } = useDictionary()
   const [isDialogOpen, setIsDialogOpen] = useState<boolean>(false)
-  const { openAllRepo } = useUpdateRepoOrPull()
+  const { openAllRepo, selectAllDependabot } = useUpdateRepoOrPull()
+  const DEPENDABOT_USER_NAME = 'jisung-lee7'
```

Comment:
put this outside of the `Page` function.

Reply:
It doesn't have to be inside of the component

Reply:
Hmm,, What do you mean,,? I can't understand.

Reply:
```typescript
const DEPENDABOT_USER_NAME = 'jisung-lee7'

export default function Page() {
  // no need to put the const variable inside the component
}
```

Reply:
Ah~ only `const DEPENDABOT_USER_NAME = 'jisung-lee7'`
I understood every const. lol.